src/odemis/acq/stream/_sync.py

```diff
@@ -151,6 +151,13 @@ def __init__(self, name, streams):
                 self._polarization = s.polarization
                 self._acquireAllPol = s.acquireAllPol
 
+        # Get integration time on detector if found in optical substream
+        self._integrationTime = None
+        for s in streams:
+            if hasattr(s, "integrationTime"):
+                # get the integration time VA
+                self._integrationTime = s.integrationTime
```

Comment:
I'd suggest that for clarity, `integrationCounts` should also be directly copied too, and they go together.